optaplanner-integration/optaplanner-spring-boot-autoconfigure/src/main/java/org/optaplanner/spring/boot/autoconfigure/SolverProperties.java

```diff
@@ -21,6 +21,8 @@
 
 public class SolverProperties {
 
+    public static final String DEFAULT_SCORE_DRL_URL = "scoreRules.drl";
```

Comment:
So are we changing this to `constraints.drl`?

Reply:
Yes, it's has been changed